packages/driver/cypress/integration/dom/visibility_spec.ts

```diff
@@ -702,6 +720,21 @@ describe('src/cypress/dom/visibility', () => {
         expect(this.$parentPosAbs.find('span')).to.be.hidden
         expect(this.$parentPosAbs.find('span')).to.not.be.visible
       })
+
+      it('is visible if position: fixed and parent has pointer-events: none', function () {
+        expect(this.$parentPointerEventsNone.find('span')).to.be.visible
+        expect(this.$parentPointerEventsNone.find('span')).to.not.be.hidden
+      })
+
+      it('is not visible if covered when position: fixed and parent has pointer-events: none', function () {
+        expect(this.$parentPointerEventsNoneCovered.find('span')).to.be.hidden
```

Comment:
I don't understand why this example is hidden - it seems it's dependent on some HTML structure outside of what's defined in `parentPointerEventsNoneCovered`? This span is visible by itself when I just add this HTML. If it's dependent on some other HTML on the page, it should probably be encapsulated into the DOM example so someone else doesn't change the outside HTML and break this test.

```html
<div style="pointer-events: none;">
  <span style="position: fixed;">parent pointer-events: none</span>
</div>
```

Reply:
Yeah it was being covered by a different element on the page, I've changed it now so that its entirely self-contained.